magento/pwa-studio
# Display page level error and disable checkout button when no available payment methods

Fixed issue #2869 Assigned by @sirugh 



## Description

Display page level error and disable checkout button when no available payment methods

## Related Issue





Closes #2869.

## Acceptance



### Verification Stakeholders

1. @sirugh 
### Specification



### Verification Steps


1. Add item to cart
2. go to checkout and complete shipping steps
3. Make sure you have payment methods enabled other than Braintree

## Screenshots / Screen Captures (if appropriate)
 
![pwa-docker localhost_8080_checkout (2)](https://user-images.githubusercontent.com/3338156/100215543-20818d00-2f37-11eb-86d7-e146b886c821.png)

![pwa-docker localhost_8080_checkout (1)](https://user-images.githubusercontent.com/3338156/100215529-1e1f3300-2f37-11eb-96cf-a9bf8f6fb0f3.png)


## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
